app/Controller/EventsController.php

```diff
@@ -2763,8 +2763,17 @@ public function freeTextImport($id) {
 		
 		if ($this->request->is('post')) {
 			App::uses('ComplexTypeTool', 'Tools');
+
+
+            Configure::write('debug', 2);
```

Comment:
Thank you for the feedback. So the best would be separated small and separated PR. It's easier for testing and review. Thank you and enjoy your evening.